runtime/src/tasks/massivethreads/tasks-massivethreads.c

```diff
@@ -619,6 +626,8 @@ char* chpl_task_idToString(char* buff, size_t size, chpl_taskID_t id) {
 // Yield.
 //
 void chpl_task_yield(void) {
+  extern void chpl_privatization_checkpoint(void);
+  chpl_privatization_checkpoint();
```

Comment:
I think you renamed it to   chpl_qsbr_checkpoint();

AFAIK Massivethreads isn't currently working with GASNet. I recommend you leave the massivethread adjustments to a separate PR (and as we know it'll have to enable task-local storage also). We don't test massivethreads nightly.

Reply:
Okay, I'll do that and remove them from massivethreads.

Reply:
Nevermind, I'll test it without GASNet to check if it works.